### ISSUE TITLE
InfluxDB 2 integration

### ISSUE THREAD

**Issue by alexanderfefelov** [pavel-odintsov/fastnetmon#900]
According to the document at https://fastnetmon.com/docs/influxdb_integration/ FastNetMon supports integration with InfluxDB due to his Graphite facility. But InfluxDB v2 does not support Graphite any more (https://docs.influxdata.com/influxdb/v2.0/upgrade/v1-to-v2/):

> ### Supported protocols
> InfluxDB 2.0 doesn’t directly support the alternate write protocols supported in InfluxDB 1.x (CollectD, Graphite, OpenTSDB, Prometheus, UDP). Use Telegraf to translate these protocols to line protocol.

Do you intend to provide native support for InfluxDB?

**Comment by pavel-odintsov** [comment]
Yep, we have plans about adding native InfluxDB protocol. Graphite is a such old legacy and we want to move away from it. But we have no ETA about it. We just finished big project to add IPv6 and we need some time to work on other things.

**Comment by pavel-odintsov** [comment]
JFYI we've started ground work to enable InfluxDB support https://github.com/pavel-odintsov/fastnetmon/commit/60d20e686ce85b2ba1e9ce355c8740ead5d8748f

**Comment by alexanderfefelov** [comment]
Thanks.

**Comment by pavel-odintsov** [comment]
https://github.com/pavel-odintsov/fastnetmon/pull/905 is ready for tests :)

**Comment by pavel-odintsov** [comment]
Just install packages from CI (they will be ready in ~3h):
```
wget https://raw.githubusercontent.com/pavel-odintsov/fastnetmon/master/src/fastnetmon_install.pl -Ofastnetmon_install.pl 
sudo perl fastnetmon_install.pl --install-ci-packages
```

And then enable InfluxDB:
```
# Before using InfluxDB you need to create database using influx tool:
# create database fastnetmon

# InfluxDB
influxdb = off
influxdb_host = 127.0.0.1
influxdb_port = 8086
influxdb_database = fastnetmon

# InfluxDB auth
influxdb_auth = off
influxdb_user = fastnetmon
influxdb_password = secure
```

**Comment by pavel-odintsov** [comment]
Let us know if you have any issues!
